### ISSUE TITLE
`curl` debugging for --data in POST requests should be JSON formatted

### ISSUE THREAD

**Issue by lenniboy** [relayr/python-sdk#12]
If a POST request with a JSON body fails the client helpfully outputs a `curl` command which helps debugging. However the data is formatted in `www-form-urlencoded` encoding rather than JSON, which is correct.

Would it be possible to change this?

**Comment by lenniboy** [comment]
Currently it posts something like
```
curl -X POST https://api.relayr.io/devices -H "Content-Type: application/json" -H "Authorization: Bearer some-token" -H "User-Agent: io.relayr.sdk.python/0.2.2 (Darwin-13.4.0; x86_64-64bit; CPython-2.7.5)" --data "owner=9210dce8-25da-4275-a73a-1383b9774255&model=175c44b5-334e-493f-8eb8-82c8cc65d29f&name=test+device&firmwareVersion=1.0.0"
```
But instead it should be something like:

curl -X POST https://api.relayr.io/devices -H "Content-Type: application/json" -H "Authorization: Bearer some-token" -H "User-Agent: io.relayr.sdk.python/0.2.2 (Darwin-13.4.0; x86_64-64bit; CPython-2.7.5)" --data '{"owner":"9210dce8-25da-4275-a73a-1383b9774255", "model":"175c44b5-334e-493f-8eb8-82c8cc65d29f", "name": "test_device","firmwareVersion": "1.0.0"}'
{"message":"model"}"

**Comment by deeplook** [comment]
Should be fixed by #06c46f7153.
